src/packages/indicator/indicator.scss

```
@@ -167,7 +195,17 @@
     .nut-indicator {
       &-line {
         width: $indicator-dot-size;
-        height: $indicator-dot-active-size;
+        height: $indicator-slider-width;
+
+        &-active {
+          transform: translateY(
+            calc(
+              var(--nutui-indicator-current, 0) /
```

Stylelint (17.12.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)